vimrc

```diff
@@ -6,9 +6,17 @@ if &shell == "/usr/bin/sudosh"
   set shell=/bin/bash
 endif
 
+if !isdirectory(expand("~/.vim/plugged"))
+  echom "Pluggins Not Installed, installing and then restart vim..."
+  autocmd VimEnter * PlugInstall --sync
+  autocmd VimEnter * qall
+endif
```

Comment:
I think this block now supersedes this line in the activate script, since I get this echo even when running `./activate.sh` now:

https://github.com/braintreeps/vim_dotfiles/blob/351010c4b8111b7c03f420bf2300b39853bc4e85/activate.sh#L14

Reply:
Fixed!